library/src/main/java/com/google/android/exoplayer2/text/eia608/Eia608Decoder.java

```diff
@@ -159,23 +175,72 @@
     0xC5, 0xE5, 0xD8, 0xF8, 0x250C, 0x2510, 0x2514, 0x2518
   };
 
+  // Maps EIA-608 PAC row numbers to WebVTT cue line settings.
+  // Adapted from: https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#x1-preamble-address-code-pac
+  private static final float[] CUE_LINE_MAP = new float[] {
+    63.33f, // Row 11
+    10.00f, // Row 1
+    26.00f, // Row 4
+    68.66f, // Row 12
+    79.33f, // Row 14
+    31.33f, // Row 5
+    42.00f, // Row 7
+    52.66f, // Row 9
+  };
+
+  // Maps EIA-608 PAC indents to WebVTT cue position values.
+  // Adapted from: https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#x1-preamble-address-code-pac
+  // Note that these cue position values may not give the intended result, unless the font size is set
+  // to allow for a maximum of 32 (or 41) characters per line.
+  private static final float[] INDENT_MAP = new float[] {
+    10.0f, // Indent 0/Column 1
+    20.0f, // Indent 4/Column 5
+    30.0f, // Indent 8/Column 9
+    40.0f, // Indent 12/Column 13
+    50.0f, // Indent 16/Column 17
+    60.0f, // Indent 20/Column 21
+    70.0f, // Indent 24/Column 25
+    80.0f, // Indent 28/Column 29
+  };
+
+  private static final int[] COLOR_MAP = new int[] {
+    Color.WHITE,
+    Color.GREEN,
+    Color.BLUE,
+    Color.CYAN,
+    Color.RED,
+    Color.YELLOW,
+    Color.MAGENTA,
+    Color.BLACK // Only used by Mid Row style changes, for PAC an value of 0x7 means italics.
+  };
+
+  // Transparency is defined in the first byte of an integer.
+  private static final int TRANSPARENCY_MASK = 0x80FFFFFF;
+
+  private static final int STYLE_ITALIC = Typeface.ITALIC;
+  private static final float DEFAULT_CUE_LINE = CUE_LINE_MAP[4]; // Row 11
```

Comment:
Thanks, good spot!

It will be fixed in the next commit, I'm currently working on displaying every line as a separate cue.